frontend/packages/console-shared/src/components/dashboard/utilization-card/TopConsumerPopover.tsx

```diff
@@ -177,21 +178,21 @@ const ConsumerItems: React.FC<ConsumerItemsProps> = React.memo(({ items, model }
       {items.map((item) => {
         const title = String(item.x);
         return (
-          <>
-            <li
-              key={title}
-              className="pf-l-flex pf-m-justify-content-space-between co-utilization-card-popover__consumer-list__item"
-              aria-labelledby="list-item1"
-            >
+          <li
+            key={title}
+            className="co-utilization-card-popover__consumer-list__item"
+            aria-labelledby="list-item1"
+          >
+            <Flex className="pf-m-justify-content-space-between">
```

Comment:
Please create a class and follow BEM instead of using flex utility classes. This is basically like inline styles and makes it difficult to use media queries to change layout.

Reply:
makes sense, `Flex` removed and created css class

Reply:
@spadgett Does it make sense here to use the flex layout and use the `[-on-breakpoint]` suffix to ensure better responsive behaviour rather than creating custom css?

Reply:
Agreed, it seems like this is what the [breakpoint modifiers](https://www.patternfly.org/v4/documentation/core/layouts/flex#documentation) are intended for.

Reply:
I agree with @spadgett.  We experimented in the 3.x console with similar attribute modifiers in the markup and found it difficult to maintain.